ndk/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 # 0.7.0 (2022-07-24)
 
+- event: Add `tool_type` getter for `Pointer`. (#323)
```

Comment:
And move it under `# Unreleased` please :)

Reply:
Sorry, i'm missing a bit of sleep